spec/index.html

```diff
@@ -1377,979 +1392,1003 @@ <h2>Reduce duplication</h2>
-        with a corresponding <a data-lt="node anchor">anchor</a>;
-        otherwise, a
-        <a data-cite="JSON-LD11-API#dom-jsonlderrorcode-loading-document-failed">loading-document-failed</a>
-        error has been detected and processing is aborted.
-      </p>
-
-      <p>
-        The <a>YAML-LD extended profile</a> allows full use of
-        <a>anchor names</a> and <a>alias nodes</a> subject to
-        the requirements described above in this section.
-      </p>
-
-      <p id="cir-extended" data-tests="">
-        If the {{JsonLdOptions/extendedYAML}} API flag is `true`, the processing result
-        will be in the <a>extended internal representation</a>.
-      </p>
-
-      <p id="aa-json-aliases" data-tests="
-      basic-manifest.html#aa-json-aliases-1-negative,
-      basic-manifest.html#aa-json-aliases-2-negative,
-      extended-manifest.html#aa-cycles-1-positive">
-        When processing using the <a>YAML-LD JSON profile</a>,
-        documents MUST NOT contain <a>alias nodes</a>;
-        otherwise, a
-        <a data-link-for="YamlLdErrorCode">profile-error</a>
-        error has been detected and processing is aborted.
-      </p>
+      <section>
+        <p>
+          Two alternative approaches have been proposed to implement the Extended profile:
+        </p>
 
-      <section id="conversion-to-ir">
-        <h2>Conversion to the Internal Representation</h2>
-
-        <p>YAML-LD processing is defined by converting YAML to the
-          <a>internal representation</a> and using [[[JSON-LD11-API]]]
-          to process on that representation,
-          after which the representation is converted back to YAML.
-          As information specific to a given <a>YAML document</a> structure is lost
-          in this transformation, much of the specifics of that
-          original representation are therefore lost in that conversion,
-          limiting the ability to fully round-trip a YAML-LD document back
-          to an equivalent representation.
-          Consequently, round-tripping in this context is limited to preservation of the semantic
-          representation of a document,
-          rather than a specific syntactic representation.</p>
-
-        <aside class="example" title="YAML Sequences">
-          <p>For example, YAML has multiple ways to encode an <a>array</a>,
-            YAML <a>block sequences</a> and
-            <a>flow sequences</a>.
-            Both forms describe the same <a>array</a> of two <a>strings</a>.</p>
-
-          <p>A YAML <a>block sequence</a>:</p>
-          <pre data-transform="updateExample"
-               data-content-type="application/yaml">
-        <!--
-          - one
-          - two
-        -->
-      </pre>
+        <ul>
+          <li>Extended Internal Representation</li>
+          <li>
+            Preprocessor to convert an Extended YAML-LD document
+            to a plain YAML-LD document
```

Comment:
Probably need to define this better. Would that be a YAML-LD document sticking to the YAML JSON profile?

Reply:
Yes, but I'd suggest we do that in another PR. Created issue #97 for that purpose.